kolibri/plugins/learn/assets/src/views/learn-page/allcontent.vue

```diff
@@ -1,6 +1,6 @@
 <template>
 
-  <card-grid :header="$tr('allContent')" ref="grid">
+  <content-cards container="grid" :contents="contentToShow" :header="$tr('allContentPageHeader')" ref="grid">
```

Comment:
Seeing as the buttons used in this component are actually links, I'm not sure what the best way to go about this would be. @rtibbles would appreciate some discussion around the best way to handle pagination and to go over some designs I thought would work